### ISSUE TITLE
Checkbox does not automatically add to subsequent lines

### ISSUE THREAD

**Issue by InterestedMind** [nvbn/everpad#112]
When using a checkbox, in the windows and android versions of Evernote, if you choose a checkbox, whenever you press enter, a checkbox is automatically populated in the next line, enabling you to continue with the next item in the checklist

In Everpad, I have to manually click the checkbox button for each line.

Please fix the behaviour to automatically insert a checkbox at a newline if the previous line has one.

Thanks!

**Comment by fmoo** [comment]
Do you need the indent functionality for this as well?  Or just the auto-insert-on-enter bit?

**Comment by fmoo** [comment]
Also of note, the first-party evernote-web client does _not_ have this feature.

**Comment by nvbn** [comment]
Maybe that must work like lists now.

**Comment by InterestedMind** [comment]
Auto insert is what I need. Indent would be a nice to have feature that
would be, er, nice to have. :-)

**Comment by thameera** [comment]
This is in fact a 'missing' feature for many who use Evernote to create to-do lists, etc. Official Windows and Android clients support this. Not present in the web client.

**Comment by InterestedMind** [comment]
Yes it does. Thanks for pointing it out.

**Comment by fmoo** [comment]
> that must work like lists now

Yeah, 90% of the current list behavior (pressing enter inserts a new list item) is handled by contenteditable=true

I am thinking would be more accurately implemented using <ul> tricks and inline css to render the checkbox outside the text
.
Intercepting <cr> while the anchor element is on the same line as a checkbox may be doable, but may turn out to be a little bit trickier.
